appveyor.bat

```diff
@@ -0,0 +1,20 @@
+set PATH=C:\Ruby200\bin;%BOOST_LIBRARYDIR%;%PATH%
+if defined MINGW_ROOT set PATH=%MINGW_ROOT%\bin;C:\msys64\usr\bin\;%PATH%
+if defined QT_DIR set PATH=%QT_DIR%\bin;%PATH%
```

Comment:
I think it might make more sense to add `%QT_DIR%` to [`CMAKE_PREFIX_PATH`](https://cmake.org/cmake/help/v3.5/variable/CMAKE_PREFIX_PATH.html). That should also remove the need for manually specifying `QT_QMAKE_EXECUTABLE`.